src/main/java/org/pac4j/jax/rs/pac4j/JaxRsProfileManager.java

```diff
@@ -131,6 +131,10 @@ public PrincipalImpl(LinkedHashMap<String, CommonProfile> profiles) {
             this.roles = Collections.unmodifiableSet(rs);
         }
 
+        public CommonProfile getProfile() {
```

Comment:
This is not related to this PR, right? I would prefer you remove it then :)

Reply:
✓